kernel_hardening_checker/checks.py

```diff
@@ -651,21 +651,34 @@ def add_sysctl_checks(l, _arch):
 # Calling the SysctlCheck class constructor:
 #   SysctlCheck(reason, decision, name, expected)
 
-    l += [SysctlCheck('self_protection', 'kspp', 'net.core.bpf_jit_harden', '2')]
+    # use an omnipresent config symbol to see if we have a config file
+    have_config_file = KconfigCheck('-', '-', 'DEFAULT_INIT', 'is present')
```

Comment:
Similar approach here: we need some kconfig option that exists in the kernel for quite a long time for all CPU architectures.

It looks like `DEFAULT_INIT` appeared in 5.8.
https://cateee.net/lkddb/web-lkddb/DEFAULT_INIT.html

Maybe we should choose something more old.

Reply:
Fixed this in https://github.com/a13xp0p0v/kernel-hardening-checker/pull/125/commits/0e0aa5f1516aeb4ffd9161d1d2c3479fe8caa39d:
changed `CONFIG_DEFAULT_INIT` to `CONFIG_LOCALVERSION`, which is older.